packages/rocketchat-katex/lib/katex

```diff
@@ -0,0 +1 @@
+../../../node_modules/katex/dist/
```

Comment:
I don't think we can use symbolic links anywhere in the project. Can you just point to the right folder where you need it?